package.json

```diff
@@ -20,7 +20,7 @@
     "db:seed": "sequelize-cli seed:generate --name",
     "db:seeds": "sequelize-cli db:seed:all",
     "db:seed:undo": "sequelize-cli db:seed:undo",
-    "test":"cross-env NODE_ENV=test npx sequelize db:migrate:undo:all && cross-env NODE_ENV=test  npm run migrate:up -all && jest --detectOpenHandles --coverage --testTimeout 100000 --forceExit",
```

Comment:
Consider not running db:migrate:undo:all before tests, that may be the reason your tests are failing because you may be deleting some migrations that are not created. you can use this script instead   
"test": "cross-env NODE_ENV=test  npm run migrate:up -all && jest --detectOpenHandles --coverage --testTimeout 100000 --forceExit"